### PR TITLE
Cherry pick/rs9116 ble and nvm delay

### PR DESCRIPTION
**Pull request by sharad-patil24**
#### Problem / Feature

Cherry-pick below commits from main branch to matter_1.0

https://github.com/SiliconLabs/sdk_support/commit/0feca55bda1af4baa76f4e30e8808321cf4e1f52
https://github.com/SiliconLabs/sdk_support/commit/da8827efde529d79e6d7f29277721ba460efbca7

### CODE REVIEW

File: matter/efr32/efr32mg12/BRD4161A/autogen/sl_event_handler.c [hunk 1]
```diff
@@ -81,6 +81,9 @@ void sl_stack_init(void)
 {
     sl_rail_util_pa_init();
     sl_rail_util_pti_init();
+#if !RS91X_BLE_ENABLE
+     sl_bt_rtos_init();
+#endif
```
Comment (mkardous-silabs) [inline matter/efr32/efr32mg12/BRD4161A/autogen/sl_event_handler.c:86]: All of the changes should not be cherry-picked. In matter 1.0 we do not use the gecko sdk freertos implementation.

File: matter/efr32/efr32mg24/BRD4186C/autogen/sl_event_handler.c [hunk 2]
```diff
@@ -75,6 +75,9 @@ void sl_service_init(void) {
 void sl_stack_init(void) {
   sl_rail_util_pa_init();
   sl_rail_util_pti_init();
+#if !RS91X_BLE_ENABLE
+  sl_bt_rtos_init();
```
Comment (mkardous-silabs) [inline matter/efr32/efr32mg24/BRD4186C/autogen/sl_event_handler.c:79]: same here

File: matter/efr32/efr32mg24/BRD4187C/autogen/sl_event_handler.c [hunk 3]
```diff
@@ -75,6 +75,9 @@ void sl_service_init(void) {
 void sl_stack_init(void) {
   sl_rail_util_pa_init();
   sl_rail_util_pti_init();
+#if !RS91X_BLE_ENABLE
+  sl_bt_rtos_init();
```
Comment (mkardous-silabs) [inline matter/efr32/efr32mg24/BRD4187C/autogen/sl_event_handler.c:79]: same here

**Review by mkardous-silabs — CHANGES_REQUESTED** [review]
A general comment, as good practice, there should be one cherry-pick PR per cherry-pick.

File: matter/efr32/efr32mg12/BRD4161A/autogen/sl_event_handler.c (hunk 1 above)
Comment (shgutte) [reply matter/efr32/efr32mg12/BRD4161A/autogen/sl_event_handler.c]: Hi @mkardous-silabs  reverted the commit

File: matter/efr32/efr32mg24/BRD4186C/autogen/sl_event_handler.c (hunk 2 above)
Comment (shgutte) [reply matter/efr32/efr32mg24/BRD4186C/autogen/sl_event_handler.c]: Hi @mkardous-silabs  reverted the commit

**Review by jmartinez-silabs — APPROVED** [review]

**Pull request merged**